doc/release-notes.md

```diff
@@ -40,36 +43,247 @@ Downgrading to a version older than 0.14.0.3 is no longer supported due to
+Chia BLS library to support migration to a new BLS signature scheme which will
+be implemented in a future version of DashCore. These changes will be made to
+align with standards and improve security.
+
+Network performance improvements
+--------------------------------
+This version of Dash Core includes multiple optimizations to the network and
+p2p message handling code.
+
+We reintroduced [Intra-Quorum Connections](https://github.com/dashpay/dips/blob/master/dip-0006.md#intra-quorum-communication)
+which were temporary disabled with the introduction of
+`SPORK_21_QUORUM_ALL_CONNECTED`. This should make communications for masternodes
+belonging to the same quorum more robust and improve network connectivity in
+general.
+
+FreeBSD and MacOS machines will now default to the new `kqueue` network mode
+which is similar to `epoll` mode on linux-based. It removes most of the CPU
+overhead caused by the sub-optimal use of `select` in the network thread when
+many connections were involved.
+
+Wallet changes
+--------------
+Upgrading a non-Hierarchical Deterministic (HD) wallet to an HD wallet is now
+possible. Previously new HD wallets could be created, but non-HD wallets could
+not be upgraded. This update will enable existing non-HD wallets to upgrade to
+take advantage of HD wallet features. Upgrades can be done via either the debug
+console or command line and a new backup must be made when this upgrade is
+performed.
+
+Users can now load and unload wallets dynamically via RPC and GUI console.
+It's now possible to have multiple wallets loaded at the same time, however only
+one of them is going to be active and receive updates or be used for sending
+trasactions at a specific point in time.
+
+Also, enabling wallet encryption no longer requires a wallet restart.
+
+Sporks
+------
+Several spork changes have been made to streamline code and improve system
+reliability. Activation of `SPORK_22_PS_MORE_PARTICIPANTS` in DashCore v0.16
+has rendered that spork unnecessary. The associated logic has been hardened
+and the spork removed. `SPORK_21_QUORUM_ALL_CONNECTED` logic has been split
+into two sporks, `SPORK_21_QUORUM_ALL_CONNECTED` and `SPORK_23_QUORUM_POSE`,
+so that masternode quorum connectivity and quorum Proof of Service (PoSe) can
+be controlled independently. Finally, `SPORK_2_INSTANTSEND_ENABLED` has a new
+mode (value: 1) that enables a smooth transition in case InstantSend needs to
+be disabled.
+
+Statoshi backport
+------------------
+This version includes a [backport](https://github.com/dashpay/dash/pull/2515)
+of [Statoshi functionality](https://github.com/jlopp/statoshi) which allows
+nodes to emit metrics to a StatsD instance. This can help node operators to
+learn more about node performance and network state in general. We added
+several command line options to give node operators more control, see options
+starting with `-stats` prefix.
+
+PrivateSend rename
```

Comment:
maybe call it a debranding?

Reply:
that's from https://blog.dash.org/updated-product-brief-dashcore-release-v0-17-339f739b5cfa :)